src/genericComponents/PrivateLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Badge as PrivateLabel } from 'react-bootstrap';
+import styled from 'styled-components';
+import { colors, fonts } from '../theme/styleVars';
+
+const Badge = () => {
+  return <CAGLabel>private</CAGLabel>;
+};
+
+const CAGLabel = styled(PrivateLabel)`
+  border-radius: 7px;
+  font-family: ${fonts.montserrat};
+  color: ${colors.secondaryFontColor};
+  letter-spacing: 0.15rem;
+  font-weight: lighter;
+  padding: 8px 10px;
+  text-transform: uppercase;
+  background-color: #b8d8c7;
```

Comment:
If a color doesn't exist in the styleVars color object for theming, we'll want to add it. We don't really have a standard for color naming, but what I like to do is go to the following website and just copy what they call it lol:

https://www.colorhexa.com/b8d8c7

<img width="435" alt="Screen Shot 2021-08-29 at 1 00 00 AM" src="https://user-images.githubusercontent.com/19152732/131240380-ecefe32d-2036-4faf-a8ef-8c39eda24bff.png">

In this case, it's a weird one: Grayish cyan - lime green. Let's just call our color variable `grayishCyan` or `grayishLightGreen` - whatever you like the most.